blocks/BrandDisplay/src/BrandDisplay.jsx

```diff
@@ -10,23 +10,23 @@ const dataSource = [
     headPic:
       'https://img.alicdn.com/tfs/TB1QMwlSXXXXXaUXXXXXXXXXXXX-122-122.png',
     pic: 'https://img.alicdn.com/tfs/TB1n6H_SXXXXXc3XpXXXXXXXXXX-616-348.png',
-    url: 'https://v.taobao.com/v/mission/case-detail?userId=38587850',
+    url: 'https://v.taobao.com/v/mission/case-detail?userId=38587850'
```

Comment:
choose another url?

Reply:
change to `https://alibaba.github.io/ice`

Reply:
done